lib/util/resolve-schema-reference.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+function resolveSchemaReference (rawSchema, ref) {
+  const resolvedReference = ref.resolve(rawSchema, { externalSchemas: [ref.definitions().definitions] })
+
+  // Ref has format `#/definitions/id`
+  const schemaId = resolvedReference?.$ref?.split('/')[2]
```

Comment:
Nit:

```suggestion
  const schemaId = resolvedReference?.$ref?.split('/', 3)[2]
```

Reply:
Oh yeah, nice catch! Thanks for the update 🎉